src/components/shared/DetailsWidget/types.ts

```diff
@@ -19,18 +19,18 @@ export const ACTION_TYPES_ICONS_MAP: {
   [ColonyActionType.SetUserRoles]: 'emoji-crane',
   [ColonyActionType.EmitDomainReputationPenalty]: 'emoji-firebolt',
   [ColonyActionType.EmitDomainReputationReward]: 'emoji-shooting-star',
-  [ColonyMotions.MintTokensMotion]: 'emoji-seed-sprout',
-  [ColonyMotions.PaymentMotion]: 'emoji-dollar-stack',
-  [ColonyMotions.MoveFundsMotion]: 'emoji-world-globe',
-  [ColonyMotions.CreateDomainMotion]: 'emoji-crane',
-  [ColonyMotions.VersionUpgradeMotion]: 'emoji-strong-person',
-  [ColonyMotions.ColonyEditMotion]: 'emoji-edit-tools',
-  [ColonyMotions.EditDomainMotion]: 'emoji-pencil-note',
-  [ColonyMotions.SetUserRolesMotion]: 'emoji-crane',
-  [ColonyMotions.EmitDomainReputationPenaltyMotion]: 'emoji-firebolt',
-  [ColonyMotions.EmitDomainReputationRewardMotion]: 'emoji-shooting-star',
-  [ColonyMotions.UnlockTokenMotion]: 'emoji-padlock',
-  [ColonyMotions.CreateDecisionMotion]: 'emoji-decisions',
-  [ColonyMotions.NullMotion]: 'forbidden-signal',
+  [ColonyActionType.MintTokensMotion]: 'emoji-seed-sprout',
+  [ColonyActionType.PaymentMotion]: 'emoji-dollar-stack',
+  [ColonyActionType.MoveFundsMotion]: 'emoji-world-globe',
+  [ColonyActionType.CreateDomainMotion]: 'emoji-crane',
+  [ColonyActionType.VersionUpgradeMotion]: 'emoji-strong-person',
+  [ColonyActionType.ColonyEditMotion]: 'emoji-edit-tools',
+  [ColonyActionType.EditDomainMotion]: 'emoji-pencil-note',
+  [ColonyActionType.SetUserRolesMotion]: 'emoji-crane',
+  [ColonyActionType.EmitDomainReputationPenaltyMotion]: 'emoji-firebolt',
+  [ColonyActionType.EmitDomainReputationRewardMotion]: 'emoji-shooting-star',
+  [ColonyActionType.UnlockTokenMotion]: 'emoji-padlock',
+  // [ColonyMotions.CreateDecisionMotion]: 'emoji-decisions',
+  // [ColonyMotions.NullMotion]: 'forbidden-signal',
```

Comment:
Null Motion was added to account for a bug in the contracts that happened at colony v9 and voting rep v5

This will still be needed for now, unless we find a better solution for it. _(something which we shouldn't tackle now)_